-code-checks

ERROR: Build errors found during clang-tidy run.
```
EventFilter/L1GlobalTriggerRawToDigi/src/L1GtTriggerMenuLiteProducer.cc:56:7: error: no matching constructor for initialization of 'const edm::ESGetToken<L1GtTriggerMenu, L1GtTriggerMaskTechTrigRcd>' [clang-diagnostic-error]
      m_l1GtMenuToken(esConsumes<L1GtTriggerMenu, L1GtTriggerMenuRcd, edm::Transition::BeginRun>()),
      ^
/cvmfs/cms-ib.cern.ch/nweek-02634/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_2_X_2020-06-24-1100/src/FWCore/Utilities/interface/ESGetToken.h:45:15: note: candidate constructor not viable: no known conversion from 'ESGetToken<[...], L1GtTriggerMenuRcd>' to 'const ESGetToken<[...], L1GtTriggerMaskTechTrigRcd>' for 1st argument
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/EventFilter/L1GlobalTriggerRawToDigi/interface/L1GtTriggerMenuLiteProducer.h
+++ b/EventFilter/L1GlobalTriggerRawToDigi/interface/L1GtTriggerMenuLiteProducer.h
@@ -117,7 +117,7 @@ private:
   const edm::ESGetToken<L1GtPrescaleFactors, L1GtPrescaleFactorsTechTrigRcd> m_l1GtPfTechToken;
   const edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskAlgoTrigRcd> m_l1GtTmAlgoToken;
   const edm::ESGetToken<L1GtTriggerMask, L1GtTriggerMaskTechTrigRcd> m_l1GtTmTechToken;
-  const edm::ESGetToken<L1GtTriggerMenu, L1GtTriggerMaskTechTrigRcd> m_l1GtMenuToken;
+  const edm::ESGetToken<L1GtTriggerMenu, L1GtTriggerMenuRcd> m_l1GtMenuToken;
 
 private:
   /// index of physics DAQ partition

